### ISSUE TITLE
[Feature Request] gocryptfs config file from another location

### ISSUE THREAD

**Issue by VladimirChalanov** [hardcore-sushi/DroidFS#243]
I would like to encrypt a folder on my sd card using gocryptfs and at the same time keep my conf file in internal storage. On Linux you can run gocryptfs with the "--config" switch to get this.

**Comment by hardcore-sushi** [comment]
Yes it would be great to add that. It could also be implemented for CryFS.

**Comment by gitoss** [comment]
+1 for this feature - I'm using DroidFS to encrypt folders on the external, unencrypted sd card because Android's built-in encryption makes the card inaccessible if the mobile phone breaks and the internal/hidden encryption key is lost.

THe internal storage is encrypted though, so it would make the config file more protected to put it there - I can keep a backup somewhere else, not carring it around in an unencryped state.
